apps/admin-portal/src/pages/external-claims.tsx

```diff
@@ -130,13 +160,26 @@ export const ExternalClaimsPage = (props): React.ReactElement => {
                     ? (
                         <ListLayout
                             advancedSearch={ <ExternalClaimsSearch onFilter={ (query) => {
-                                getExternalClaims(null, null, null, query);
+                                //getExternalClaims(null, null, null, query);
```

Comment:
Can we remove this commented line ? If it's something we will use later. Add a TODO